styles/Vocab/Plone/reject.txt

```diff
@@ -0,0 +1,5 @@
+node
+nodejs
+javscript
+js
+Javascript
```

Comment:
I think these should be sorted alphanumerically, and case-insensitive.

Reply:
same as above.